lua/apisix/plugin.lua

```diff
@@ -281,53 +281,43 @@ end
 
 
 local function merge_service_route(service_conf, route_conf)
-    local new_service_conf
+    local new_conf = core.table.deepcopy(service_conf)
```

Comment:
no test cases.

Reply:
I think the old test cases already cover this change.